util/reggen/window.py

```diff
@@ -2,13 +2,14 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-from typing import Dict, List, Optional
+from typing import Dict, List
 
 from .access import SWAccess
 from .lib import (check_keys, check_str, check_bool, check_int,
                   expand_parameter)
 
 REQUIRED_FIELDS = {
+    'name': ['s', "Name of the window"],
```

Comment:
for consistency with the fields below:

```suggestion
    'name': ['s', "name of the window"],
```

Reply:
Oops, good point. Thanks!